Samples/Deploy-AzureDataFactory/README.md

```diff
@@ -0,0 +1,33 @@
+# Deploy-AzureDataFactory
+Deploys an Azure Data Factory (ADF) from the command line. 
+
+
+## Synopsis
+The script only deploys linked services, datasets and pipelines for a data factory, no current suport for Gateways.	
+For this script to run correctly, in the PowerShell context in which this is called, you should have already loged in
+to your Azure account and selected the correct subscription, e.g.:
+```
+	Login-AzureRmAccount
+	Get-AzureRmSubscription �SubscriptionName �YourSubscription� | Select-AzureRmSubscription
```

Comment:
change these quotes to normal/un-stylized double quotes so that copy-pasting this line would work in the console